.mailmap

```diff
@@ -5,6 +5,6 @@ Brian Gianforcaro <bgianf@serenityos.org> <b.gianfo@gmail.com>
 Max Wipfli <max.wipfli@serenityos.org> <mail@maxwipfli.ch>
 Sam Atkins <atkinssj@serenityos.org> <atkinssj@gmail.com>
 Karol Kosek <krkk@serenityos.org> <krkk@krkk.ct8.pl>
-David Tuin <davidot@serenityos.org> <david.tuin@gmail.com>
+davidot <davidot@serenityos.org> David Tuin <david.tuin@gmail.com>
```

Comment:
The mailmap in master works fine though?
`git log` lists your commits as "David Tuin <davidot@serenityos.org>" already, note that I don't think github respects .mailmap if that's what you're looking at.

Reply:
I just noticed that when I ran `git summary` it had both a `David Tuin` and `davidot` entry. But strangely enough most other things seem to pick the currently entry up fine :shrug:

Reply:
The old entry said "Find all commits made by anyone with email address `<david.tuin@gmail.com>` and map them to 'David Tuin <davidot@serenityos.org>'".

The new entry says "Find all commits made by anyone with email address `<david.tuin@gmail.com>` _and_ name 'David Tuin' and map them to 'davidot <davidot@serenityos.org>'".

Is that what you want? The new entry should match fewer or equal as many entries as the new one.